webrtc/simplecall.html

```diff
@@ -97,9 +99,11 @@
     gFirstConnection.addIceCandidate(event.candidate);
   });
 
-  var onRemoteStream = test.step_func(function(event) {
+  var onRemoteTrack = test.step_func(function(event) {
     var videoTag = document.getElementById('remote-view');
-    videoTag.srcObject = event.stream;
+    if (!videoTag.srcObject) {
```

Comment:
yes. This is the easiest way to avoid setting srcObject again. Maybe the more correct one is to do
```if (videoTag.srcObject !== stream) { ... }```
?